jerry-core/api/jerry-snapshot.c

```diff
@@ -1047,11 +1047,10 @@ jerry_exec_snapshot (const uint32_t *snapshot_p, /**< snapshot */
  */
 static void
 scan_snapshot_functions (const uint8_t *buffer_p, /**< snapshot buffer start */
-                         const uint8_t *buffer_end_p, /**< snapshot buffer end */
```

Comment:
Hm, I don't really like removing this, but since this is a static function we can reintroduce it later if needed.

Reply:
Let me hm, too. I didn't want to suggest the removal of this argument, although I see how my comment has led here. I should dive a bit deeper into snapshot functions to check the semantics behind `buffer_end_p` and `literal_base_p`, where and when and how they are used, why they are the same at all/most call sites, how can the literal data start after the snapshot buffer, etc. If they are really always equal then that may hint at a conceptual problem (or that may be too harsh, something that could be optimised, or named better, etc).

So I +1 now to no removal and to switch back to the original redundant arguments. (But someone could put this on a todo list and take a closer look.)